tests/test_geometry.py

```diff
@@ -922,7 +922,7 @@ def test_3d_point_converted_to_2d_point():
 
 
 def test_crs():
-    CRS = geometry.CRS
+    CRS = geometry.CRS  # noqa: N806
```

Comment:
`from geometry import CRS`

... would be a clear way to do this without needing a `noqa` comment, but it looks like we only use it as `geometry.CRS` below anyway, which means we can drop the line entirely?